recipe/get_license_family.py

```diff
@@ -0,0 +1,38 @@
+import license_expression
```

Comment:
Smithy depends on this so the import is fine, but you probably want to add it to the deps directly here so that if we ever remove it from smithy, this feedstock will still build.

Reply:
looks like smithy is not installed here as I had assumed so you need to list the dep.